core/src/main/java/io/grpc/inprocess/InProcessTransport.java

```diff
@@ -78,6 +80,9 @@
 
   public InProcessTransport(String name) {
     this.name = name;
+    this.serverStreamAttributes = Attributes.newBuilder()
+        .set(ServerCall.REMOTE_ADDR_KEY, new GrpcUtil.InProcessSocketAddress(name))
```

Comment:
Not sure if `GrpcUtil` is the best place for it but couldn't find better and didn't want to create a separate `Util` class just for it. Alternatively, we can put it into this class  (`InProcessTransport`).